addon/ng2/commands/doc.js

```diff
@@ -0,0 +1,25 @@
+/* jshint node: true */
```

Comment:
Could you make this a `.ts` file?  We support TypeScript in angular-cli. After that you can remove the `use strict` below as all ts files are strict.